tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js

```diff
@@ -46,7 +46,7 @@ context('Multiple users. Assign task, job.', () => {
 
     describe(`Testing case "${caseId}"`, () => {
         // First user is "admin".
-        it('Register second user and logout.', () => {
+        it('Register second user, tries to create task and logout.', () => {
```

Comment:
Probably it would be better to do it in different file because the current has other features tested. Also is not obvious to find these test cases here according to the filename.

Reply:
I added these actions in test because if they will run in different file then verify take many time